plugins/obs-webrtc/webrtc/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "webrtc-rs-lib"
+version = "0.1.0"
+edition = "2021"
+build = "build.rs"
+
+[dependencies]
+webrtc = "0.6.0"
+webrtc-mdns = "0.5.2"
+tokio = { version = "1.16.1", features = ["full"] }
```

Comment:
are you sure that pulling in the `full` feature set of `tokio` is necessary?

Reply:
Some of this was inherited, I'll trim it

Reply:
Fixed in 4c891626cae971dc2fc5ffd4f51658fee715094c